docs/daily-usage.md

```diff
@@ -376,6 +384,69 @@ services:
 
 </details>
 
+## Visual Studio Code Integration
+
+Generated projects have out-of-the-box integration with
+[Visual Studio Code](https://code.visualstudio.com/) to provide you with a better and
+easier-to-use experience through:
+
+- Suggested extensions
+- Launch configurations (See section on `tasks` file)
+- Workspace definition (See section on `tasks` file)
+- Integration with invoke (See section on `tasks` file)
+- Debugger integration (See section on `tasks` file and sections below)
+
+### Debugging Python code
+
+In order to debug Python code, you need to:
+
+1. Install suggested extensions, as mentioned above
+2. Run the project with the provided `invoke` commands. (This will automatically
+   generate the VSCode configurations for you) **Note**: As the configurations are
+   defined per project and depending on the Odoo addons you are working on, you might
+   need to update your configuration frequently with `invoke write-code-workspace-file`
+   (done automatically when you do `invoke start`, for example).
+3. Go to the "Run" tab in VSCode, select "Start Odoo and debug Python", and click "Run".
+   The program will stop in any breakpoint you have and you will be able to analyze your
+   Python environment.
+
+### Debugging JS code
+
+In order to debug JavaScript code, you need to:
+
+1. Install suggested extensions, as mentioned above
+2. Run the project with the provided `invoke` commands. (This will automatically
+   generate the VSCode configurations for you) **Note**: As the configurations are
+   defined per project and depending on the Odoo addons you are working on, you might
+   need to update your configuration frequently with `invoke write-code-workspace-file`
+   (done automatically when you do `invoke start`, for example).
+3. Go to the "Run" tab in VSCode, select "Start Odoo and debug JS in [Firefox/Chrome]",
```

Comment:
Always use 1 please, to avoid confusion if some points are added or removed in between:

```suggestion
1. Go to the "Run" tab in VSCode, select "Start Odoo and debug JS in [Firefox/Chrome]",
```